securebanking-openbanking-uk-rs-obie-api/src/main/java/com/forgerock/securebanking/openbanking/uk/rs/api/obie/payment/v3_1_4/domesticpayments/DomesticPaymentsApi.java

```diff
@@ -87,6 +87,9 @@
             @ApiParam(value = "Indicates the user-agent that the PSU is using.")
             @RequestHeader(value = "x-customer-user-agent", required = false) String xCustomerUserAgent,
 
+            @ApiParam(value = "Indicates if 'ReadRefundAccount' was set to 'Yes' or 'No' in the consent. Defaults to 'No'", required = false)
+            @RequestHeader(value = "x-read-refund-account", required = false) String xReadRefundAccount,
```

Comment:
This value is stored against the consent, so we will need to provide it to the simulator in some way.

Reply:
That seems like a really good solution to me.